packages/cli/src/commands/init.ts

```diff
@@ -201,7 +201,19 @@ const resolveOptionsWithPrompts = async (
     }
 
     if (!options.endpointSlug) {
-      resolvedOptions.endpointSlug = await promptEndpointSlug(path);
+      const resolvedPath = resolvePath(options.projectPath);
+      const packageJSONPath = pathModule.join(resolvedPath, "package.json");
```

Comment:
This looks good! Almost ready to merge, just need to use the passed in `path` argument to the `resolveOptionsWithPrompts` function as it's already the resolvedPath, so there is no need to call `resolvePath` again.

Reply:
Awesome! I'll tend to this now.

Reply:
I realized that this PR has a conflict. So I synced my forked repo with the main one and pulled. 

When I did `git status` I saw that a lot of files have been modified/created. Should I just abandon this PR and create a new one? 

![image](https://github.com/triggerdotdev/trigger.dev/assets/48127194/d35d73c1-081f-4144-a0ce-cf57dfb19dc7)

cc: @ericallam